### ISSUE TITLE
Error in example dataset deprecation

### ISSUE THREAD

**Issue by peterdesmet** [adokter/bioRad#103]
```R
plot(VP) # similar for plot(VPTS) and plot(SCAN)
```

Results in:

```
Error in example_vp(...) : could not find function "example_vp"
In addition: Warning message:
'x' is deprecated.
Use 'example_vp' instead.
See help("Deprecated")
```

These data are listed as deprecated functions, but they are not functions:

https://github.com/adokter/bioRad/blob/1741aa022ecdbfda8cdf0ea9e548e80f6b201b58/R/bioRad-deprecated.R#L393-L403

**Comment by peterdesmet** [comment]
Suggest to update to:

```R
#' @export
VP <- function(...) {
  .Deprecated(msg = "Example object of class 'vp' has been renamed from 'VP' to 'example_vp'.")
}
```

So we get for `plot(VP)`:

```R
Error in curve(expr = x, from = from, to = to, xlim = xlim, ylab = ylab,  : 
  'expr' did not evaluate to an object of length 'n'
In addition: Warning message:
Example object of class 'vp' has been renamed from 'VP' to 'example_vp'. 
```

@adokter, ok?

**Comment by adokter** [comment]
Fixed the bug in commit d32277548d3ea8cfea657ffb63cc9f62820b4f27 (for now) by a `NULL` function. Above example won't work, e.g. just executing `VP` will not give you the object as before. We could also just have all data double, or we might be able to make an alias with a hook https://stat.ethz.ch/R-manual/R-patched/library/base/html/userhooks.html (not sure though this will work, not tested yet).

**Comment by adokter** [comment]
I think very few people will have code with explicit use of bioRad's example objects, so no need to deprecate in more detail. closing
